### PR TITLE
[example] Redesigned footer and CUBE css methodology

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Summary | Résumé
This PR contains the redesigned footer as a test to tr out a new css methodology: [CUBE](https://cube.fyi/)

tl;dr of cube:
- CUBE stands for compositions-utilities-blocks-exceptions
- The utilities are automatically generated from a tokens file using a small sass library called gorko, and looks a lot like tailwind, but with our own tokens.
- Compositions, blocks and exceptions are a lot like bem, but are more designed to work with the css cascade instead of against it.

In this PR, I also pull the WET css file into the sass pipeline. I don't know if that's a good idea, but my long term objective would be to remove it. 

I use the data-system attribute to only target one part of the website at a time, so that we can slowly build our cube css structure. 

kind of like this:
``` scss
@import "current";

[data-system] {
  @import "cube";
}
```

then like this:
```html
<footer data-system="cube-cds" role="contentinfo">
things
</footer>
```

# Why am I proposing this?

I think we'd have a better experience if we had more control over our styles, and if we all aligned onto a methodology. I know we like tailwind, and cube is kinda like tailwind, but only the good parts, plus we get to customize the tokens and make it part of our sass pipeline.

It can get frustrating to fight overrides and disorganized css. This approach is mild in my opinion and shouldn't break things as we start building our new site.  

# Test instructions | Instructions pour tester la modification

1. Did something break?

# Help requested | Aide requise

If you want to try out this methodology, take this branch and try to implement one other design. See if you enjoy it!
It will probably be easier than you think as the redesign was also made into a [mini cube design system](https://digital-canada-system.netlify.app/). Most utilities, blocks and compositions are already coded. 

# Reviewer checklist | Liste de vérification du réviseur

This is a suggested checklist of questions reviewers might ask during their
review | Voici une suggestion de liste de vérification comprenant des questions
que les réviseurs pourraient poser pendant leur examen :

- [ ] Does this meet a user need? | Est-ce que ça répond à un besoin utilisateur?
- [ ] Is it accessible? | Est-ce que c’est accessible?
- [ ] Is it translated between both offical languages? | Est-ce dans les deux
      langues officielles?
- [ ] Is the code maintainable? | Est-ce que le code peut être maintenu?
- [ ] Have you tested it? | L’avez-vous testé?
- [ ] Are there automated tests? | Y a-t-il des tests automatisés?
- [ ] Does this cause automated test coverage to drop? | Est-ce que ça entraîne
      une baisse de la quantité de code couvert par les tests automatisés?
- [ ] Does this break existing functionality? | Est-ce que ça brise une
      fonctionnalité existante?
- [ ] Should this be split into smaller PRs to decrease change risk? | Est-ce
      que ça devrait être divisé en de plus petites demandes de tirage (« pull
      requests ») afin de réduire le risque lié aux modifications?
- [ ] Does this change the privacy policy? | Est-ce que ça entraîne une
      modification de la politique de confidentialité?
- [ ] Does this introduce any security concerns? | Est-ce que ça introduit des
      préoccupations liées à la sécurité?
- [ ] Does this significantly alter performance? | Est-ce que ça modifie de
      façon importante la performance?
- [ ] What is the risk level of using added dependencies? | Quel est le degré de
      risque d’utiliser des dépendances ajoutées?
- [ ] Should any documentation be updated as a result of this? (i.e. README
      setup, etc.) | Faudra-t-il mettre à jour la documentation à la suite de ce
      changement (fichier README, etc.)?

### CODE REVIEW

**Comment by dj2** [comment]
This seems like a large and foundational change. Can we please open a bug to discuss before landing this? It sounds like CUBE is very new, still changing, are we going to have to keep iterating and changing as CUBE evolves? What's the benefit of this over going through and cleaning up or existing styles?

Why is this better than tailwind (which we also don't use on the website I don't think).

**Comment by amazingphilippe** [comment]
Is there a way to let circleci know that there is a new dependency?

**Comment by amazingphilippe** [comment]
Sure, I didn't mean to force this onto anyone, it was just to let people see how it could work. CUBE isn't changing, its only a way to look at how we write css. We don't use tailwind, but I know we like it a lot. 

The benefit of CUBE is that we can evolve things without getting stuck in css specificity.

**Comment by dj2** [comment]
Sorry, I didn't mean for you to close this, having it open as an example is awesome. I'd just _also_ like to discuss in a bug somewhere so we can understand what we're dealing with.
